salt/states/group.py

```diff
@@ -275,21 +275,31 @@ def present(name,
     return ret
 
 
-def absent(name):
+def absent(name, onlyifempty=False):
```

Comment:
Can we change this name to be more pythonic/stateful? At the very least some underscores could be added, `only_if_empty`, but there's probably a better stateful name we could choose. But I am not coming up with anything good. What do you think @terminalmage?

Reply:
I think `only_if_empty` is probably best here. We just need to make sure that all references to the new variable name are updated in the code below.

Reply:
Didn't really like the name either, but took one to move on. Very open for suggestions.